src/components/plot/plot.scss

```diff
@@ -48,4 +48,11 @@
 
 .copy-button {
   cursor: pointer;
+  position: relative;
+}
+
+.copied {
```

Comment:
That said, I'd say keep the styleName but remove the position/left/margin below.

Instead, let's make the font-size a bit smaller.  It's quite too big currently.